src/pages/blog/javascript-debugging-slightly-beyond-consolelog/index.mdx

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Javascript debugging [slightly] beyond console.log"
-description: ""
+description: "My favourite tool for javascript debugging is `console.log`. I mean, I love `console.log`. If I ever get a programming tattoo, it's going to be `console.log` or some variation thereof. Simple to use, works every time, and every programming language I've used since I was 9 has had some variation of it."
```

Comment:
Can you add this tweet after the first paragraph?

https://twitter.com/Swizec/status/1417914800743485440